style.css

```diff
@@ -0,0 +1,102 @@
+.wrapper {
+	width: 80%;
+	margin: auto;
+	border: dashed 1px red;
+}
+
+.header {
+	display: flex;
+	justify-content: space-between;
+}
+
+.header__logo {
+	padding: 40px 0;
+	font-family: 'ptserif-bold', arial, helvetica, Sans-serif;
+	font-weight: bold;
+	font-size: 24px;
+	line-height: 31px;
+	position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)